rekt.go

```diff
@@ -169,6 +225,113 @@ func muteHandler(caller *discordgo.Member, msg *discordgo.Message, args []string
 	return nil
 }
 
+func unmuteHandler(caller *discordgo.Member, msg *discordgo.Message, args []string) (err error) {
```

Comment:
Should this also lock the unmute goroutine? (Someone tell me how to lock stuff in go lol)